### DISCUSSION TITLE
"Peak mode implementation"?

### DISCUSSION THREAD

**Discussion by stevelitchfield** [Archfx/duo-de#133] [General]
I'm sorry to be thick, but what does this mean, from the latest changelog? Peak what?
Thanks.

**Comment by Archfx** [comment]
Hi, @stevelitchfield "Peak mode implementation" is a proof of concept implementation of the notification panel when the device is partially unfolded (let's say the hinge angle is about 40 degrees).  

In our current implementation, you have to unlock the device using the fingerprint while the device is unfolded less than 50 degrees to see it. It is just a proof of concept to show that it is possible to have this feature similar to the original A12L. The layout and other features are not finalized yet.

**Reply by stevelitchfield** [reply]
Ah, so you mean 'peek' mode. Different word, different spelling!! 8-)

**Reply by Archfx** [reply]
Ops, thanks for pointing it out. I didn't notice it and used it everywhere. Let me fix it!
